### PR TITLE
feat(mojaloop/#3264): tls cert reload issue

### PR DESCRIPTION
**Pull request by vijayg10**
feat(mojaloop/#3264): tls cert reload issue

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -215,6 +215,67 @@ jobs:
...
+      # - run:
+      #     name: Build docker-compose
+      #     command: |
+      #       docker-compose build
+      - run:
+          name: Generate certificates
+          command: |
+            cd docker/haproxy/tls
+            sh createSecrets.sh
+            cd ../../../
```
Comment (mdebarros) [inline .circleci/config.yml:243]:
```suggestion
            cd $CIRCLE_WORKING_DIRECTORY
```

File: modules/api-svc/src/index.js [hunk 2]
```diff
@@ -210,7 +210,8 @@ class Server extends EventEmitter {
             await this.wso2.auth.start();
         }
 
-        const updateInboundServer = !_.isEqual(this.conf.inbound, newConf.inbound);
+        const updateInboundServer = !_.isEqual(this.conf.inbound, newConf.inbound)
+        || !_.isEqual(this.conf.outbound, newConf.outbound);
```
Comment (mdebarros) [inline modules/api-svc/src/index.js:214]:
```suggestion
            || !_.isEqual(this.conf.outbound, newConf.outbound);
```

**Review by mdebarros — COMMENTED** [review]
See my comments

File: modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            partySubIdOrType: null,
+        },
+    },
+    timestamp: Date.now(),
+    headers: [],
+    });
+
+    await new Promise(resolve => setTimeout(resolve, 10000));
+    await producer.sendDomainEvent(partyInfoRequestedDmEvt);
+    await new Promise(resolve => setTimeout(resolve, 10000));
```
[the remark this answers is not in the document]
Comment (vijayg10) [reply modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js:112]: Thanks @mdebarros . Created a story for this https://github.com/mojaloop/project/issues/3408

File: modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            partyIdType: 'MSISDN',
+            partyIdentifier: '1234567890',
+            partySubIdOrType: null,
+        },
+    },
+    timestamp: Date.now(),
+    headers: [],
+    });
+
+    await new Promise(resolve => setTimeout(resolve, 10000));
```
[the remark this answers is not in the document]
Comment (vijayg10) [reply modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js:110]: Done

File: modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    await producer.sendDomainEvent(partyInfoRequestedDmEvt);
+    await new Promise(resolve => setTimeout(resolve, 10000));
```
[the remark this answers is not in the document]
Comment (vijayg10) [reply modules/api-svc/test/integration-pm4ml/FSPIOPEventHandler/partyInfoRequest.test.js:151]: Thanks. Added a TODO comment and the story is https://github.com/mojaloop/project/issues/3407

File: .circleci/config.yml (hunk 1 above)
Comment (vijayg10) [reply .circleci/config.yml:243]: Thanks, Not committing this here and I handled this manually to individual extra CI job runs.

File: modules/api-svc/src/index.js (hunk 2 above)
Comment (vijayg10) [reply modules/api-svc/src/index.js:214]: Thanks, Not committing this here and I handled this manually to individual extra CI job runs.

**Review by mdebarros — APPROVED** [review]
+1

**Pull request merged**